scss/_base/_forms.scss

```diff
@@ -1,59 +1,99 @@
+// A label and a form element.
+//
+// .-padded - Adds extra padding above and below field group.
+// .-inline - Form elements appear side-by-side.
+//
+// Styleguide Forms - Field Groups
+.field-group {
+  margin-bottom: ($base-spacing / 2);
```

Comment:
I'd argue for `margin-bottom`, since pretty much all forms will end with a `.form-actions` (submit, etc) block, and we can account for the expected spacing there. With `margin-top` any container with a form would need to have less top padding to account for the extra margin added by the first field item/row.